AT 04.02 | Folder > Rename Folder

Kononirin opened the pull request:
https://trello.com/c/gXpAr9po

NataliaRamanenka commented:
you need to add a link to the AT card in trello into the description of PR
and name of PR should be the same as name of AT card in trello

Kononirin commented:
> you need to add a link to the AT card in trello into the description of PR and name of PR should be the same as name of AT card in trello

Hello, Natalia. Thank you for review. I changed name of PR and added link to the Trello card.
